cockatrice/src/carddatabase.cpp

```diff
@@ -194,23 +196,29 @@ void SetList::markAllAsKnown()
 
 void SetList::guessSortKeys()
 {
-    // sort by release date DESC; invalid dates to the bottom.
-    QDate distantFuture(2050, 1, 1);
-    int aHundredYears = 36500;
+    defaultSort();
     for (int i = 0; i < size(); ++i) {
         CardSetPtr set = at(i);
-        if (set.isNull()) {
```

Comment:
this null check might still be necessary

Reply:
Good catch, got lost in the shuffle. Fixed.